locales/en-US/strings.properties

```diff
@@ -72,6 +72,7 @@ search_settings=Change Search Settings
 # LOCALIZATION NOTE (section_info_option): This is the screenreader text for the
 # (?) icon that would show a section's description with optional feedback link.
 section_info_option=Info
+section_info_privacy_policy=Privacy Policy
```

Comment:
Looks like it'll be "Privacy Notice"

Reply:
yeah, changing this to section_info_privacy_notice = Privacy Notice